private-templates-service/client/src/components/Common/UnpublishModal/UnpublishModal.tsx

```diff
@@ -66,19 +71,19 @@ class UnpublishModal extends React.Component<Props> {
       <BackDrop>
         <Modal>
           <Header>Unpublish Template</Header>
-    <Description style={{marginBottom: 0}}>{STRINGS.UNPUBLISH_CONFIRMATION}<DescriptionAccent>{template.name} - {this.props.templateVersion}</DescriptionAccent>?</Description>
+          <Description style={{ marginBottom: 0 }}>{STRINGS.UNPUBLISH_CONFIRMATION}<DescriptionAccent>{template.name} - {this.props.templateVersion}</DescriptionAccent>?</Description>
           <Description>{STRINGS.UNPUBLISH_WARNING}</Description>
           <CenterPanelWrapper>
             <Container>
               <ACWrapper>
-                <AdaptiveCard cardtemplate={template} templateVersion={"1.0"}/>
+                <AdaptiveCard cardtemplate={template} templateVersion={"1.0"} />
               </ACWrapper>
               <TemplateName>{template.name}</TemplateName>
             </Container>
           </CenterPanelWrapper>
           <BottomRow>
             <ButtonGroup>
-              <CancelButton text="Cancel" onClick={this.props.toggleModal} />
+              <CancelButton text="Cancel" onClick={() => { this.props.closeModal() }} />
```

Comment:
why `() => {this.props.closeModal()}`
instead of 
`onClick={this.props.closeModal}`